modin/pandas/base.py

```diff
@@ -3856,6 +3856,40 @@ def values(self):  # noqa: RT01, D200
         """
         return self.to_numpy()
 
+    def _repartition(self, axis: Optional[int] = None):
```

Comment:
There is no sense to pass the axis for Series so I think the method should be separate for DataFrame and Series.